test/e2e_tests/dir_test.go

```diff
@@ -527,7 +527,12 @@ func (suite *dirTestSuite) TestGitStash() {
 }
 
 func (suite *dirTestSuite) TestReadDirLink() {
-	if runtime.GOOS == "windows" || suite.adlsTest && strings.ToLower(enableSymlinkADLS) != "true" {
+	// Symbolic link creation requires admin rights on Windows.
```

Comment:
This doesn't change functionality, but is really a fixup for another change I made.
I had added the OS check to make one long conditional, even though it was meant for the ADLS flag.
So this change separates them, along with a better print.